add-ons/istio/README.adoc

```diff
@@ -0,0 +1,84 @@
+:linkattrs:
+
+= Istio Add-on
+
+This add-on deploys https://Istio.io/[Istio] service mesh on to Minishift-provisioned OpenShift cluster.
+
+== Minishift profile setup
+
+The standard Istio deployment creates few pods and its related resources. 
```

Comment:
I feel like we're missing some context in this sentence. The phrase "creates few pods", in particular, seems misleading -- it implies that Istio only uses a few pods, which sounds like few resources will be required. It would be more useful to detail how many pods are created than to say "few".

Also, for more than one pod, it would be "their related resources" rather than "its related resources".

Reply:
rephrased a bit